modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -748,6 +748,8 @@
     #include <nrfx_config_nrf54h20_enga_application.h>
```

Comment:
nit: drop `and CMake injected definitions` 67c5c07db0962c08ceb412c7715888d3925464c0 as it is done later